modules/social_features/social_group/modules/social_group_secret/config/install/group.content_type.secret_group-group_node-event.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - group.type.secret_group
+    - node.type.event
+  module:
+    - gnode
+    - node
+id: secret_group-group_node-event
+label: event
+description: 'Adds <em class="placeholder">Event</em> content to groups both publicly and privately.'
```

Comment:
This description probably is copy-pasted and needs to be edited. It currently refers to "publicly and privately", however content in this group will not be publicly available, correct?

Reply:
@NatashaSchon what text needs to use here?

Reply:
I might need some more context here. Is this a description that says this event content will be added to certain groups and will be visible both privately and publicly? Or to public and private groups? 

Suggestions (if the content is not publicly available): 
Adds 'Event' content to groups using private content settings. 
Adds 'Event' content to private groups. 

@OH17L